src/main/java/org/spongepowered/common/util/BookFaker.java

```diff
@@ -58,7 +58,10 @@ public static void fakeBookView(BookView bookView, Player player) {
         receiver.sendPacket(new SPacketSetSlot(WINDOW_PLAYER_INVENTORY, bookSlot, ItemStackUtil.toNative(item)));
 
         // Next we tell the client to open the Book GUI
-        receiver.sendPacket(new SPacketCustomPayload("MC|BOpen", new PacketBuffer(Unpooled.buffer())));
+        // 0 for the main hand, 1 for the offhand
+        PacketBuffer packetbuffer = new PacketBuffer(Unpooled.buffer());
+        packetbuffer.writeInt(0);
```

Comment:
You should use `packetbuffer.writeEnumValue(EnumHand.MAIN_HAND)` here. There is no need to hard code the internal protocol representation, MC uses `writeEnumValue` for opening books.